sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubTestClient.java

```diff
@@ -38,7 +38,7 @@
  * testing {@link #publish}, {@link #pull}, {@link #acknowledge} and {@link #modifyAckDeadline}
  * methods. Relies on statics to mimic the Pubsub service, though we try to hide that.
  */
-class PubsubTestClient extends PubsubClient implements Serializable {
+public class PubsubTestClient extends PubsubClient implements Serializable {
```

Comment:
I have one request, which is to put the `@Experimental` tag on this. It may not be very visible to users, but we definitely don't want a test utility to have backwards-compatibility expectations, since we didn't really plan it that carefully.